src/main/java/duke/storage/Storage.java

```diff
@@ -0,0 +1,54 @@
+package duke.storage;
+
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.List;
+
+import duke.task.Task;
+
+/**
+ * Represents the storage for the tasklist.
+ */
+public class Storage {
+    private static final Path path = Paths.get("data", "duke.txt");
```

Comment:
private final constants should be uppercase
```suggestion
    private static final Path PATH = Paths.get("data", "duke.txt");
```